packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js

```diff
@@ -3,12 +3,17 @@ import { Box, Paragraph } from 'grommet'
 import { inject, observer } from 'mobx-react'
 import PropTypes from 'prop-types'
 import React from 'react'
-import { ThemeProvider } from 'styled-components'
+import styled, { ThemeProvider } from 'styled-components'
 
 import getTaskComponent from './helpers/getTaskComponent'
 import TaskHelp from './components/TaskHelp'
 import { default as TaskNavButtons } from './components/TaskNavButtons'
 
+const StyledBox = styled(Box)`
+  max-height: 600px;
```

Comment:
Isn't this just going to cause content clipping if we have a task with lots of choices?

Reply:
Ah yes. I meant to add an overflow here.

Reply:
I realized this was a bad idea in my comment: https://github.com/zooniverse/front-end-monorepo/issues/561#issuecomment-474437971

Volunteers might miss tasks this way. It's should be a min-height instead that allows the container to grow.

Reply:
Gotcha. I was thinking we were using a max-height and encouraging users to stick to max task numbers with overflow as a safety net. I've pushed a new commit removing max height. The container should grow as needed.

Reply:
We can maybe do that once the workflow editor page is redesigned, but that's a ways off still.